### DISCUSSION TITLE
Gothic 1 - combat movement simplified but limited

### DISCUSSION THREAD

**Discussion by 0xF16** [Try/OpenGothic#453] [Q&A] [answered]
I remember in vanilla Gothic 1 I was able to hit the opponent from the top, left or right. Possibly there was a way to do the block (hardly remember) in OpenGothic I cannot do that. I only hit one way. Is it a bug or I remember it wrong? Cannot kill a meat bug with current attack xD

I don't know if that makes a difference, but I play on MacOS

**Comment by Try** [comment]
Hi, @0xF16 !

It doesn't seem right indeed: I've give a run for G1-OpenGothic today on my windows-pc:
`forward` + `action` - forward attack
`lest`/`right` + `action` - side attack
`back` + `action` - block attack

**Reply by 0xF16** [reply]
Thank you for confirming. Do you want me to report an issue for it then @Try?

**Reply by ghost** [reply]
Hey @Try, I was gonna ask if there is a way to change that mechanic - I thought that's just the current state of things. I wonder why it is different in your setup and ours?

In both G1 and G2 pressing Ctrl makes the hero automatically attack, which makes fights more difficult and blocking seems possible only while taking a step back.

**Reply by Try** [reply, marked as answer]
> In both G1 and G2 pressing Ctrl makes

Not in G2. G2 has 2 different control schemes. Maybe you need to enable G1-controls ?
![изображение](https://github.com/Try/OpenGothic/assets/122894/d5fa65e6-a73b-4541-9163-87c3ef74e86f)

**Reply by ghost** [reply]
Yes, that works for Gothic 2. No luck for Gothic 1 - for some reason combat works like in G2, it seems that there's just no menu option to assign side attacks and block. It is not a priority for me, though, just letting you know. The game is playable anyway.

**Reply by Try** [reply]
Checkout 0039c277 (or just latest) commit - G1 controls should be default option in general; and only option for G1

**Reply by ghost** [reply]
Perfect. 👍
